src/main/java/tech/ypsilon/bbbot/database/MongoController.java

```diff
@@ -7,40 +7,53 @@
 import com.mongodb.client.MongoClients;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
+import lombok.Getter;
 import org.bson.Document;
 import org.bson.codecs.configuration.CodecRegistries;
 import org.bson.codecs.configuration.CodecRegistry;
 import tech.ypsilon.bbbot.BotInfo;
+import tech.ypsilon.bbbot.ButterBrot;
+import tech.ypsilon.bbbot.config.MongoSubconfig;
+import tech.ypsilon.bbbot.util.GenericController;
 import tech.ypsilon.bbbot.database.codecs.BirthdayCodec;
 import tech.ypsilon.bbbot.database.codecs.DirectoryCodec;
 import tech.ypsilon.bbbot.database.codecs.LinkCodec;
 import tech.ypsilon.bbbot.database.codecs.StudyGroupCodec;
 import tech.ypsilon.bbbot.database.codecs.VerificationCodec;
-import tech.ypsilon.bbbot.settings.SettingsController;
+import tech.ypsilon.bbbot.util.Initializable;
 
 import java.util.Collections;
 
-public class MongoController {
+public class MongoController extends GenericController implements Initializable {
 
     private static MongoController instance;
 
-    private final MongoClient CLIENT;
-    private final MongoDatabase DATABASE;
+    private MongoClient CLIENT;
+    private MongoDatabase DATABASE;
+
+    private @Getter boolean disabled = false;
```

Comment:
What exactly is the point of `disabled`? Did you implement a feature to disable all commands that are using the database?

Reply:
The commands are not disabled, they just don't work if the mongo connection was not established